src/main/scala/com/agar/core/context/DefaultAgarSystem.scala

```diff
@@ -4,6 +4,6 @@ import scala.concurrent.duration._
 
 object DefaultAgarSystem extends AgarSystem {
 
-  override def timeout(): FiniteDuration = 100 millis
+  override def timeout(): FiniteDuration = 1 second
```

Comment:
That'll be very slow and the client will not have a great refresh rate.

Reply:
Agree I change it

Reply:
The definition is unused now. The current timeout used is set up to 200 milliseconds.

Reply:
Kep